custom_components/tuya_local/devices/ballu_aura.yaml

```diff
@@ -0,0 +1,105 @@
+name: Ballu Aura
+products:
+  - id: r9lwbruavixdspeb
+    name: "Ballu Aura  9/12"
+primary_entity:
+  entity: climate
+  dps:
+    - id: 1
+      name: hvac_mode
+      type: boolean
+      mapping:
+        - dps_val: false
+          value: "off"
+        - dps_val: true
+          constraint: mode
+          conditions:
+            - dps_val: "COOL"
+              value: cool
+            - dps_val: "DRY"
+              value: dry
+            - dps_val: "FAN"
+              value: fan_only
+    - id: 2
+      name: temperature
+      type: integer
+      range:
+        min: 17
+        max: 30
+      unit: C
+    - id: 3
+      name: current_temperature
+      type: integer
+    - id: 4
+      name: mode
+      type: string
+      hidden: true
+    - id: 5
+      name: fan_mode
+      type: string
+      mapping:
+        - dps_val: "1"
+          value: low
+        - dps_val: "2"
+          value: medium
+        - dps_val: "3"
+          value: high
+    - id: 19
+      name: temperature_unit
+      type: string
+    - id: 104
+      name: swing_mode 
+      type: boolean
+      mapping:
+        - dps_val: true
+          value: vertical
+        - dps_val: false
+          value: "off"
+    
+secondary_entities:
+  - entity: select
+    translation_key: temperature_unit
+    category: config
+    dps:
+      - id: 19
+        name: option
+        type: string
+        mapping:
+          - dps_val: C
+            value: celsius
+          - dps_val: F
+            value: fahrenheit
+  - entity: binary_sensor
+    translation_key: defrost
+    category: diagnostic
+    dps:
+      - id: 102        
+        type: boolean 
+        name: sensor
+  - entity: number
+    category: config
+    translation_key: timer
+    dps:
+      - id: 105
+        name: value 
+        type: integer 
+        unit: m
+        range:
+          min: 0
+          max: 7200
+
+  - entity: sensor
+    category: diagnostic
+    name: Alarm
+    dps:
+      - id: 106
+        name: sensor
+        type: integer 
+  - entity: sensor
+    category: diagnostic
+    name: model
```

Comment:
model should just be an attribute on the climate entity. It is not useful as a sensor I think (it will never change)